packages/widget/src/widget.ts

```diff
@@ -73,6 +85,9 @@ class SygmaProtocolWidget extends BaseComponent {
           </section>
         </form>
         <section class="poweredBy">${sygmaLogo} Powered by Sygma</section>
+        <sygma-overlay-component
```

Comment:
Should we distinguish between the internal components and the top-level ones? For example: overlay, dropdown, and button are used internally. Maybe we can name with sygma prefix the components like network or amount selector as it's used as a top-level import and the "helper" components without sygma? wdyt? 😃

Reply:
Aside from overlay, do we have another component that doesn't have the basic layout being used -either button or dropdown- coded in the same component itself? I agree with the idea but I wonder if we would need to tackle small changes in the way the use the top level import components. Just to have a better understanding

Reply:
@Lykhoyda even internal components exists in global namespace. In case our widget is used in another app, that registered "overlay" component with the same name, we could clash and our widget might render their component instead of ours.